tools/spectral/Makefile

```
@@ -0,0 +1,32 @@
+# A Self-Documenting Makefile: http://marmelab.com/blog/2016/02/29/auto-documented-makefile.html
+
+SPECTRAL_VERSION=6.11.1
```

what about using the version defined in an env variable if defined otherwise default to this?
```suggestion
SPECTRAL_VERSION=${SPECTRAL_VERSION:-6.11.1}
```